client/jetpack-onboarding/main.jsx

```diff
@@ -0,0 +1,56 @@
+/** @format */
+/**
+ * External dependencies
+ */
+import React from 'react';
+import PropTypes from 'prop-types';
+import { compact } from 'lodash';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Wizard from 'components/wizard';
+import {
+	JETPACK_ONBOARDING_COMPONENTS as COMPONENTS,
+	JETPACK_ONBOARDING_STEPS as STEPS,
+} from './constants';
+
+class JetpackOnboardingMain extends React.Component {
+	static propTypes = {
+		stepName: PropTypes.string,
+	};
+
+	static defaultProps = {
+		stepName: STEPS.SITE_TITLE,
+	};
+
+	render() {
+		const { stepName, steps } = this.props;
+
+		return (
+			<Wizard
+				basePath="/jetpack/onboarding"
+				components={ COMPONENTS }
+				steps={ steps }
+				stepName={ stepName }
+			/>
+		);
+	}
+}
+
+export default connect( () => {
+	// Note: here we can select which steps to display, based on user's input
+	const steps = compact( [
```

Comment:
Why the `compact`? We're in charge of defining these constants, so there shouldn't be any falsy ones anway, no?

Reply:
Ah, good question. 

The thing is that some steps might be enabled, others might not be enabled. This, in some cases, will depend on the user input (which we're planning to store in Redux). That's the rationale behind putting those in the `mapStateToProps`, essentially.

For example, the business address step will be shown only if the user has selected the Business site type. So for example, here we'll have:

```
getJetpackOnboardingSiteType( state ) === 'business' && STEPS.BUSINESS_ADDRESS
```

Let me know if that still doesn't make any sense.

Reply:
I figured that was the reason, but I'd go with YAGNI for now. Easy enough to add that `compact` later.